### PR TITLE
Update publishing script with setup and canary files

### PR DESCRIPTION
**Pull request by TimoWilken**
The new script also pulls changes to ali-bot on every iteration, and loops faster (sleeping only a minute between iterations).

This is to simplify the aurora service definition.

### CODE REVIEW

**Comment by ktf** [comment]
Sorry, can we keep it easy and simply have the commands we need to run one after the other? What is the advantage of the looping?

**Comment by TimoWilken** [comment]
@ktf is this better?

**Comment by TimoWilken** [comment]
I'll have to change this if #972 is merged, so marking as draft for now.

File: publish/run-all-rpms.sh [hunk 1]
```diff
@@ -1,7 +1,47 @@
...
+# look for the RPMs they expect.
+# We build this file list now as aliPublish can take some time, and other builds
+# (that we don't want to notify yet) might finish in the meantime. We run
+# aliPublish only for one specific architecture, so we have
+# architecture-specific canary files under rpmstatus/$arch/.
+s3cmd ls "s3://alibuild-repo/rpmstatus/$arch/" | cut -b 32- > canaries.txt
+
+case "$arch" in
+  el8.*)
+    pip install boto3    # aliPublishS3 needs boto3.
```
Comment (ktf) [inline publish/run-all-rpms.sh:31]: Can you put the dependency in `setup.py`?

**Review by ktf — APPROVED** [review]

**Comment by ktf** [comment]
We deploy tomorrow.

File: publish/run-all-rpms.sh (hunk 1 above)
Comment (TimoWilken) [reply publish/run-all-rpms.sh:31]: It's already there. However, `setup.py` (and `requirements.txt`) also has stuff we don't need here, like `python-ldap`, `PyGithub` and `klein`. If you prefer, I can put a `pip install -r ../requirements.txt` in the script instead, though.

Comment (TimoWilken) [reply publish/run-all-rpms.sh:31]: (We don't install ali-bot here, we just clone it, as we need the config files from it, and if we're cloning it already, we might as well just use the scripts from the repo.)

**Pull request merged**